packages/server/api/controllers/orderItems.controller.js

```diff
@@ -0,0 +1,48 @@
+const knex = require('../../config/db');
+
+// add movie to the cart
+let createdID = 0;
+const addMovieToCart = async (body) => {
+  await knex('order_items').insert({
+    order_id: body.order_id,
+    movie_id: body.movie_id,
+  });
+
+  createdID += 1;
```

Comment:
Have you tested if created ID will work correctly if you restart the server? It might be switched back to 0, which is not what we want ideally. 

If not, try to think how we could read the last created ID from the database instead. 

You might even be able to use some built-in logic from the database, where the ID is auto-generated and auto-incremented.

Reply:
Yes, you are right, it restart the number after refresh the server 
I will have a look at it but would be happy if you can provided some resource that I want read about this :)

Reply:
@babak-f  Yes, you are right, it restart the number after refresh the server I will have a look at it but would be happy if you can provided some resources that I can read about this :)

Reply:
I have fixed my code now :) 
I have changed order_items addMovieToCart from post request to put request now.

order_id in order_items table is not auto_increment and not unique itself because one order_id can have multiple movies so this table is used to link between orders and movies table 

After I changed to put, I return order_id and movie_id